### DISCUSSION TITLE
Delete workflow run

### DISCUSSION THREAD

**Discussion by yyovkov** [community/community#25921] [Actions]
<p>After cleaning all old runs from Actions, the workflow is not visible anymore in the list of workflows. In order to see the workflow again, the files in “.github/workflows” directory should be renamed. Seems the filename is ignored. How this filenames can be removed from “ignore list”.?</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/yyovkov">@yyovkov</a>,</p>
<p>Glad to see you in Github Community Forum!</p>
<p>This is by designed. Once you delete all the workflow runs, the workflow will be invisible from the list.<br>
Rename the workflow yaml file will not show it again.</p>
<p>However, you can trigger the workflow so that it will appear in the list again. For example, if you only use ‘pull_request’ event in the yaml, create a pull request to trigger the workflow.</p>
<p>Thanks</p>
